lib/modules/platform/scm-manager/scm-manager-helper.ts

```diff
@@ -0,0 +1,174 @@
+import { ScmManagerHttp } from '../../../util/http/scm-manager';
+import {
+  DefaultBranchSchema,
+  PagedPullRequestSchema,
+  PagedRepoSchema,
+  PullRequestSchema,
+  RepoSchema,
+  UserSchema,
+} from './schema';
+import type { Link, PullRequest, Repo, User } from './schema';
+import type { PullRequestCreateParams, PullRequestUpdateParams } from './types';
+
+let token: string;
+export const setToken = (newToken: string): void => {
+  token = newToken;
+};
```

Comment:
don't store the token here. it should be passed from initPlatform or it's coming from hostrules automatically after platform init.